vterm.el

```diff
@@ -490,26 +488,31 @@ Argument EVENT process event."
     (run-hook-with-args 'vterm-exit-functions
                         (if (buffer-live-p buf) buf nil))))
 
-(defun vterm--window-size-change-26 (frame)
-  "Callback triggered by a size change of the FRAME.
-
-This is only used, when variable `emacs-version' < 27. Calls
-`vterm--window-size-change' for every window of FRAME."
-  (dolist (window (window-list frame))
-    (vterm--window-size-change window)))
-
-(defun vterm--window-size-change (window)
-  "Callback triggered by a size change of the WINDOW.
-
-Feeds the size change to the virtual terminal."
-  (with-current-buffer (window-buffer window)
-    (when (and (processp vterm--process)
-               (process-live-p vterm--process))
-      (let ((height (window-body-height window))
-            (width (window-body-width window))
-            (inhibit-read-only t))
-        (set-process-window-size vterm--process height width)
-        (vterm--set-size vterm--term height width)))))
+(defun vterm--window-adjust-process-window-size(process windows)
```

Comment:
Why not use `display-line-numbers-update-width`. The cursor should be anyway at the end of the buffer. We should just think about, how it interacts with `vterm-copy-mode`.